app/controllers/excursions_controller.rb

```diff
@@ -1,30 +1,27 @@
 class ExcursionsController < ApplicationController
-  def new
-  end
+  def new; end
 
   def create
-    params_2 = {
+    {
       title: params[:title],
       description: params[:description],
       location: params[:location]
     }
     #post "backend.herokuapp.com/users/#{current_user.id}/excursions?#{params_2.to_query}"
```

Comment:
Good call, yes this is unused for now. Be weary for the team starting to use the Faraday calls, and to name Line 5 as some variable to pass into Line 10 :)